jobbergate-cli/poetry.lock

```diff
@@ -1,9 +1,10 @@
-# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.4.1 and should not be changed by hand.
```

Comment:
I see no change in dependencies on pyproject.toml. Could you revert the changes in this file?

Reply:
Rebased, removed, and force-pushed in commit a5b7946.